docs/language/src/builtins/widgets.md

```diff
@@ -438,12 +438,17 @@ Same as [`ListView`](#listview), and in addition:
 -   **`current-sort-column`** (_out_ _int_): Indicates the sorted column. -1 mean no column is sorted.
 -   **`columns`** (_in-out_ _\[[`TableColumn`](structs.md#tablecolumn)\]_): Defines the model of the table columns.
 -   **`rows`** (_\[\[[`StandardListViewItem`](structs.md#standardlistviewitem)\]\]_): Defines the model of table rows.
+-   **`current-row`** (_in-out_ _int_): The index of the currently active row. -1 mean none is selected, which is the default.
 
 ### Callbacks
 
 -   **`sort-ascending(`_`int`_`)`**: Emitted if the model should be sorted by the given column in ascending order.
 -   **`sort-descending(`_`int`_`)`**: Emitted if the model should be sorted by the given column in descending order.
 
+### Functions
+
+-   **`set-current-row(_index: int_)`: Sets the current row and brings it into view
```

Comment:
```suggestion
-   **`set-current-row(index: int)`: Sets the current row and brings it into view
```

btw, the one in StandardListView is also wrong:
https://slint-ui.com/releases/1.0.2/docs/slint/src/builtins/widgets#functions